### PR TITLE
[Jupyter] Remove dev_server from tests

### PR DESCRIPTION
**Pull request by bbonenfant**
I converted all the tests away from using the dev_server fixture and removed the fixture.

Additionally:
* I added back in the python tests that were skipped because of the missing chunk errors. Doing so required a couple of changes to the names of the resources within the tests.
* I updated the download code a bit to download to the jupyter's `root_dir` as opposed to the cwd of the server. I think the "correct" approach anyways, since you can configure which directory the jupyter server serves files from, but it also has the added benefit of allowing us to configure this value during the test so that we can write files to a temp directory. The way this test was written, you needed to delete the downloaded files before running the test again.
* I also split up the config and pps/publish tests to their own files.

### CODE REVIEW

File: jupyter-extension/jupyterlab_pachyderm/tests/test_config.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+from tornado.web import Application
+
+from pachyderm_sdk.config import ConfigFile
+
+from jupyterlab_pachyderm.env import PACH_CONFIG
+
+
+@pytest.mark.no_config
+async def test_config_no_file(app: Application, http_client: AsyncClient):
+    """Test that if there is no config file present, the extension does not
```
Comment (kevyang) [inline jupyter-extension/jupyterlab_pachyderm/tests/test_config.py:15]: TIOLI: to me this test seems to more test that the server correctly returns an `INVALID` cluster status rather than testing this negative statement

File: jupyter-extension/jupyterlab_pachyderm/tests/test_config.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    # Act
+    response = await http_client.put("/config", json=data)
+
+    # Assert
+    response.raise_for_status()
+    payload = response.json()
+    assert payload["cluster_status"] == "INVALID"
+    assert payload["pachd_address"] == invalid_address
+    assert app.settings.get("pachyderm_client") is None
```
Comment (kevyang) [inline jupyter-extension/jupyterlab_pachyderm/tests/test_config.py:47]: can we also check that the config file was not written to?

**Review by kevyang — APPROVED** [review]
lgtm, a few minor comments

File: jupyter-extension/jupyterlab_pachyderm/tests/test_config.py (hunk 1 above)
Comment (bbonenfant) [reply jupyter-extension/jupyterlab_pachyderm/tests/test_config.py:15]: Yeah I think there's some truth to what you're saying. Really its testing that no default config is set (pachd_address == "") and that the cluster status is INVALID. I'll update the docstring to better document this

File: jupyter-extension/jupyterlab_pachyderm/tests/test_config.py (hunk 2 above)
Comment (bbonenfant) [reply jupyter-extension/jupyterlab_pachyderm/tests/test_config.py:47]: great point, I'll add this
